rb/lib/selenium/webdriver/chrome/options.rb

```diff
@@ -40,33 +43,58 @@ class Options
         # @option opts [Array<String>] :extensions A list of paths to (.crx) Chrome extensions to install on startup
         # @option opts [Hash] :options A hash for raw options
         # @option opts [Hash] :emulation A hash for raw emulation options
+        # @option opts [Hash] :local_state A hash for the Local State file in the user data folder
+        # @option opts [Boolean] :detach whether browser is closed when the driver is sent the quit command
+        # @option opts [String] :debugger_address address of a Chrome debugger server to connect to
+        # @option opts [Array<String>] :exclude_switches command line switches to exclude
+        # @option opts [String] :minidump_path Directory to store Chrome minidumps (linux only)
+        # @option opts [Hash] :perf_logging_prefs A hash for performance logging preferences
+        # @option opts [Array<String>] :window_types A list of window types to appear in the list of window handles
         #
 
         def initialize(**opts)
+          opts.delete(:browser_name)
+          opts[:browser_name] = 'chrome'
+
           @args = Set.new(opts.delete(:args) || [])
           @binary = opts.delete(:binary) || Chrome.path
           @prefs = opts.delete(:prefs) || {}
+
+          @encoded_extensions = opts.delete(:encoded_extensions) || []
```

Comment:
`Array(opts.delete(:encoded_extensions))` would make this slightly more flexible (allow passing one extension or array of extensions) while still defaulting [] if nil passed